client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -273,7 +280,7 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
     }
   }
 
-  def getUserIdentifier(): UserIdentifier = {
+  def getUserIdentifier: UserIdentifier = {
```

Comment:
Seems we must delete parentheses in dollar expressions in string literal to be consist with the method definition
```
   // RssShuffleFallbackPolicyRunner.scala
   if (!available) {
      logWarning(s"Quota exceed for current user ${lifecycleManager.getUserIdentifier}.")
    }
```

Reply:
> Seems we must delete parentheses in dollar expressions in string literal to be consist with the method definition
> 
> ```
>    // RssShuffleFallbackPolicyRunner.scala
>    if (!available) {
>       logWarning(s"Quota exceed for current user ${lifecycleManager.getUserIdentifier}.")
>     }
> ```

Done.